openmetadata-ui/src/main/resources/ui/src/components/LineageProvider/LineageProvider.tsx

```diff
@@ -0,0 +1,958 @@
+          ...pre,
+          nodes: updatedNodes,
+          edges: updatedEdges,
+        };
+      });
+    },
+    [nodes, edges, entityLineage]
+  );
+
+  useEffect(() => {
+    if (entityFqn) {
+      fetchLineageData(entityFqn, lineageConfig);
+    }
+  }, [lineageConfig, entityFqn, queryFilter]);
+
+  useEffect(() => {
+    const updatedNodes = createNodes(
+      entityLineage.nodes ?? [],
+      entityLineage.edges ?? []
+    );
+    const updatedEdges = createEdges(
+      entityLineage.nodes ?? [],
+      entityLineage.edges ?? []
+    );
+
+    setNodes(updatedNodes);
+    setEdges(updatedEdges);
+  }, [entityLineage]);
+
+  const activityFeedContextValues = useMemo(() => {
+    return {
+      isDrawerOpen,
+      loading,
+      isEditMode,
+      nodes,
+      edges,
+      reactFlowInstance,
+      entityLineage,
+      lineageConfig,
+      selectedNode,
+      selectedColumn,
+      zoomValue,
+      status,
+      expandedNodes,
+      tracedNodes,
+      tracedColumns,
+      expandAllColumns,
+      onInitReactFlow,
+      onPaneClick,
+      onConnect,
+      onNodeDrop,
+      onNodeCollapse,
+      onColumnClick,
+      onNodesChange,
+      onEdgesChange,
+      onQueryFilterUpdate,
+      onZoomUpdate,
+      onDrawerClose,
+      toggleColumnView,
+      loadChildNodesHandler,
+      fetchLineageData,
+      removeNodeHandler,
+      onNodeClick,
+      onEdgeClick,
+      onColumnEdgeRemove,
+      onLineageConfigUpdate,
+      onLineageEditClick,
+      onAddPipelineClick,
+    };
+  }, [
+    isDrawerOpen,
+    loading,
+    isEditMode,
+    nodes,
+    edges,
+    entityLineage,
+    reactFlowInstance,
+    lineageConfig,
+    selectedNode,
+    selectedColumn,
+    zoomValue,
+    status,
+    expandedNodes,
+    tracedNodes,
+    tracedColumns,
+    expandAllColumns,
+    onInitReactFlow,
+    onPaneClick,
+    onConnect,
+    onNodeDrop,
+    onNodeCollapse,
+    onColumnClick,
+    onQueryFilterUpdate,
+    onNodesChange,
+    onEdgesChange,
+    onZoomUpdate,
+    onDrawerClose,
+    loadChildNodesHandler,
+    fetchLineageData,
+    toggleColumnView,
+    removeNodeHandler,
+    onNodeClick,
+    onEdgeClick,
+    onColumnEdgeRemove,
+    onLineageConfigUpdate,
+    onLineageEditClick,
+    onAddPipelineClick,
+  ]);
+
+  return (
+    <LineageContext.Provider value={activityFeedContextValues}>
+      {children}
+      <EntityLineageSidebar newAddedNode={newAddedNode} show={isEditMode} />
+
+      {isDrawerOpen &&
+        !isEditMode &&
+        (selectedEdge ? (
+          <EdgeInfoDrawer
+            hasEditAccess
+            edge={selectedEdge} // handle this access
+            nodes={nodes}
+            visible={isDrawerOpen}
+            onClose={() => {
+              setIsDrawerOpen(false);
+              setSelectedEdge(undefined);
+            }}
+            onEdgeDetailsUpdate={onEdgeDetailsUpdate}
+          />
+        ) : (
+          <EntityInfoDrawer
+            isMainNode={false}
```

Comment:
It will be always false?

Reply:
It was in use previously. Now for all cases it's false.
Have cleaned up all references now.